### ISSUE TITLE
Convert inches to cm in Extent field of museum objects

### ISSUE THREAD

**Issue by Gabz73** [sciencehistory/scihist_digicoll#2656]
Museum objects use inches instead of cm in Past Perfect. We want to convert the extent statements to metric. Can this be done on the back end?  Conversion s/b 2.54 cm per inch, with one decimal point retained.

**Comment by eddierubeiz** [comment]
This is perfectly feasible, but it's not a 10-minute job either.

See e.g.: https://digital.sciencehistory.org/works/fb4948977 for an example: all the dimensions are contained in one string.
So we want a recipe that can take a few hundred strings of the form:
`10.25 in. H x 8.25 in.` (each of which can contain up to three separate dimensions in inches)
and return a string with each value converted appropriately, and of course with the new units.

**Comment by eddierubeiz** [comment]
(Just pointing out it's a small but very interesting puzzle.)

**Comment by jrochkind** [comment]
Yep. i'd use a regex to look for something like `/(\d+\.?\d*) in\./`.  and just replace each one with `${converted_to_cm} cm.`

Does need some testing to make sure no data corruption, it is a bit risky true!  Still a human doing it manually would probably make some errors too!

**Comment by Gabz73** [comment]
Record one decimal in cm measurement

**Comment by eddierubeiz** [comment]
Oh, we want cm. measurements to one decimal.

**Comment by eddierubeiz** [comment]
[Fixed - new results attached.](https://github.com/user-attachments/files/21105066/july_7_staging.txt)

**Comment by eddierubeiz** [comment]
Just in case, here's the tab-separated [list of changes to museum extents I made in production today](https://github.com/user-attachments/files/21145250/production_metric_conversions.txt).

**Comment by eddierubeiz** [comment]
Conversions are done in production.
